cmd/quickstart.go

```diff
@@ -354,19 +360,23 @@ func retryWithSampleSpec(ctx context.Context, workflowFile *workflow.Workflow, i
 	return true, err
 }
 
-func shouldLaunchStudio(ctx context.Context, wf *run.Workflow) bool {
-	// Only one source at a time is supported in the studio at the moment
-	if len(wf.SourceResults) == 1 {
-		// If the source has a linting result, then it was loaded successfully so we can show something in the studio
-		if maps.Values(wf.SourceResults)[0].LintResult != nil {
-			// This will be true in most cases for admins, unless they have `auth switch`ed to a different workspace
-			if config.GetWorkspaceID() == "self" {
-				return true
-			}
-		}
+func shouldLaunchStudio(ctx context.Context, wf *run.Workflow, fromQuickstart bool) bool {
+	// TODO: Remove this when ready to launch for everyone
+	if !config.IsAdminUnsafe() {
+		return false
 	}
 
-	return false
+	// Only one source at a time is supported in the studio at the moment
+	// If the source has a linting result then it was loaded successfully, so we can show something in the studio
+	sourceResults := maps.Values(wf.SourceResults)
+	canLaunch := len(sourceResults) == 1 && sourceResults[0].LintResult != nil
+
+	shouldLaunch := fromQuickstart || !config.SeenStudio() || config.IsAdminUnsafe()
```

Comment:
I don't think we should have different behaviour for admins once launched. Can we prompt at the end with a yes/no?

Reply:
I see what you mean